ecs-cli/modules/cli/compose/project/project.go

```diff
@@ -38,6 +38,7 @@ type Project interface {
 	Context() *context.ECSContext
 	ServiceConfigs() *config.ServiceConfigs
```

Comment:
we are removing this in an upcoming PR, yes?

Reply:
jes